tests/test_connection_reuse.py

```
@@ -0,0 +1,107 @@
+import os
+
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)